lib/platform/azure/index.ts

```diff
@@ -113,7 +113,8 @@ export async function getRepos(): Promise<string[]> {
 
 export async function getRawFile(
   fileName: string,
-  repoName?: string
+  repoName?: string,
+  branch?: string | null
```

Comment:
```suggestion
  branch?: string
```

Should match interface declaration